lib/ui/painting/image_encoding.cc

```diff
@@ -199,21 +202,21 @@ Dart_Handle EncodeImage(CanvasImage* canvas_image,
   return Dart_Null();
 }
 
-sk_sp<SkData> EncodeImage(const sk_sp<SkImage>& raster_image,
-                          ImageByteFormat format) {
+fml::StatusOr<sk_sp<SkData>> EncodeImage(const sk_sp<SkImage>& raster_image,
+                                         ImageByteFormat format) {
   TRACE_EVENT0("flutter", __FUNCTION__);
 
   if (!raster_image) {
-    return nullptr;
+    return fml::Status(fml::StatusCode::kInternal, "Missing raster image.");
   }
 
   switch (format) {
     case kPNG: {
       auto png_image = SkPngEncoder::Encode(nullptr, raster_image.get(), {});
 
       if (png_image == nullptr) {
-        FML_LOG(ERROR) << "Could not convert raster image to PNG.";
```

Comment:
Wait, do we want to maintain these logs somewhere?

Reply:
These strings get returned to the dart:ui layer and printed there as an error.